test/integration/narrow/routes/vet-visits.test.js

```diff
@@ -0,0 +1,35 @@
+const { getFarmerApplyData } = require('../../../../app/session')
+const {
+  getLatestApplicationsBySbi
+} = require('../../../../app/api-requests/application-api')
+
+jest.mock('../../../../app/session')
+jest.mock('../../../../app/api-requests/application-api')
+
+describe('Claim vet-visits', () => {
+  test('GET /claim/vet-visits route returns 200', async () => {
+    const options = {
+      method: 'GET',
+      url: '/claim/vet-visits',
+      auth: false
+    }
```

Comment:
Will fail once auth is enabled.
A test for failure when no auth provided
A test for 200 when auth is provided - also check for content on page

Reply:
Thanks for your comment. Done!